service/src/java/org/apache/hive/service/auth/AuthenticationProviderFactory.java

```diff
@@ -76,6 +77,9 @@ public static PasswdAuthenticationProvider getAuthenticationProvider(AuthMethods
       return new CustomAuthenticationProviderImpl((conf == null) ? AuthMethods.CUSTOM.getConf() : conf);
     } else if (authMethod == AuthMethods.NONE) {
       return new AnonymousAuthenticationProviderImpl();
+    } else if (authMethod == AuthMethods.SAML) {
+      //TODO right thing to do?
+      return new AnonymousAuthenticationProviderImpl();
```

Comment:
not sure I understand how this works.
When auth is set to SAML on server, is this provider even called?

Reply:
I don't think it is necessary. I removed this from the PR.